frontend/src/views/SubjectList.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="main-content subject-list">
+    <AppBreadCrumb/>
+    <div class="subject-content">
+      <div class="card-container">
```

Comment:
Same deal with the above -- could utilise space better on larger screen widths.

Reply:
agree